### PR TITLE
docs: fix linter errors for accessibility.md

### PR DESCRIPTION
**Pull request by dsnoeijer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #47179

## What is the new behavior?
No more linter errors in the file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/accessibility.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ You should avoid situations where focus returns to the `body` element after a ro
 
 ### Active links identification
 
-CSS classes applied to active `RouterLink` elements (usually via `RouterLinkActive`) provide a visual indication regarding as to which link is currently active. Such indication doesn't apply to blind or visually impaired users, in order to provide such information the `aria-current` attribute should be applied to the element as well (for more information see [MDN aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current)).
+CSS classes applied to active `RouterLink` elements (usually via `RouterLinkActive`) provide a visual guide to which link is currently active. This guide doesn't apply to blind or visually impaired users. To provide such information the `aria-current` attribute should be applied to the element as well (for more information see [Mozilla Developer Network (MDN) aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current)).
```
Comment (bob-watson) [inline aio/content/guide/accessibility.md:130]: This is a pretty dense sentence. I tried to split the ideas out here. If this conveys the same information, you can use this change. If not, please revise as required.
```suggestion
CSS classes applied to active `RouterLink` elements, such as `RouterLinkActive`, provide a visual cue to identify the active link.
Unfortunately, a visual cue doesn't help blind or visually impaired users.
Applying the `aria-current` attribute to the element can help identify the active link.
For more information, see [Mozilla Developer Network (MDN) aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current)).
```

File: aio/content/guide/accessibility.md [hunk 2]
```diff
@@ -153,7 +153,9 @@ The following example shows how to apply the `active-page` class to active links
     </nav>
 ```
 
-## Additional resources
+<!-- vale Angular.Angular_Spelling = NO -->
+
+## Extra resources
```
Comment (bob-watson) [inline aio/content/guide/accessibility.md:158]:
```suggestion
## More information
```

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for the edit.
This is a complex topic with challenging text.
I added some suggestions for you to review.
Thanks!

**Comment by dsnoeijer** [comment]
@bob-watson Done.

**Review by bob-watson — APPROVED** [review]
LGTM

reviewed-for: global-docs-approvers

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
Caretaker note: this PR was globally approved, so we can proceed with the merge.

**Pull request closed without merging**